horace_core/GUI/horace.m

```diff
@@ -1968,7 +1967,7 @@ function Rebin_rebin_pushbutton_Callback(hObject, eventdata, handles)
         %must strip out square brackets, if user has inserted them:
         s1=strfind(lostephi,'['); s2=strfind(lostephi,']');
```

Comment:
Isn't this chunk handled by `read_vector`? If not, why not?

Reply:
this code may process bunch of brackets -- it splits the string into multiple strings separated by brackets. And read_vector deals with each peace. May be better solution is possible, but, again, not so keen on doing changes which currently can not be unit tested.  Job for a future GUI improvements